Task 5/Task 5/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("5.1\n");
+            Console.WriteLine("Введите текст капсом на русском языке");
+            string text = Console.ReadLine();
```

Comment:
Зачем такие строгие требования? Вы же можете сами (программно) перевести все символы в верхний регистр
```suggestion
            Console.WriteLine("Введите текст на русском языке");
            string text = Console.ReadLine().ToUpper();
```